src/globals/theme/_ics-theme.scss

```diff
@@ -34,6 +34,25 @@ $support-04: #3c6df0 !default; // ultramarine-50
 $input-border: 1px solid $ui-04 !default;
 $input-label-weight: 500 !default;
 
+// Border Radius
```

Comment:
This is amazing!

One suggestion is to add this to a separate file, within the same folder - I'm thinking about the context in which someone may swap out the standard theme variables with say, Watson theme, and because these theme variables are missing, it would break some components' styles.

What do you think?